tests/quantity/test_storage.py

```diff
@@ -91,6 +91,9 @@ def test_modifying_numpy_data_modifies_view():
     assert quantity.view[0, 0] == 1
     assert quantity.view[2, 2] == 5
     assert quantity.view[4, 4] == 3
+    assert quantity.field[0, 0] == 1
+    assert quantity.field[2, 2] == 5
+    assert quantity.field[4, 4] == 3
     assert quantity.data[0, 0] == 1
     assert quantity.data[2, 2] == 5
     assert quantity.data[4, 4] == 3
```

Comment:
Maybe someday it will make sense to do this with a halo and we can test that the data is different than the field or view

Reply:
Your wishes are my command*.


*Subject to budget constraint, time and implementation technical details. Offer can be removed at any time for no reasons. Definition of "wishes" and "command" subject to change.